src/components/2021/number.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
```

Comment:
Does this constrain font sizes within a min and max? Seems cool

Reply:
It _should_ yes. Been a while since I’ve used [FitText](http://fittextjs.com) on project and it works mostly well.